account_bank_statement_import/models/account_bank_statement_import.py

```diff
@@ -132,7 +160,7 @@ def _import_statement(self, stmt_vals):
         return self._create_bank_statement(stmt_vals)
 
     @api.model
-    def _parse_file(self, data_file):
+    def _parse_file(self, dummy_data_file):
```

Comment:
@hbrunn I added the dummy_ prefix because the parameter is not used in this skeleton function. And then pylint complains about unused parameters. Can we leave it? Otherwise I will rename it - at the cost of pylint complaining.